test/views/ViewSpecBase.scala

```diff
@@ -75,13 +85,19 @@ trait ViewSpecBase extends SpecBase {
   def assertContainsText(doc: Document, text: String): Assertion =
     assert(doc.toString.contains(text), "\n\ntext " + text + " was not rendered on the page.\n")
 
+  def assertNotContainsText(doc: Document, text: String): Assertion =
+    assert(!doc.toString.contains(text), "\n\ntext " + text + " was not rendered on the page.\n")
```

Comment:
Shouldn't this be `was rendered on the page` instead of `was not rendered on the page`?